src/style.css

```diff
@@ -0,0 +1,116 @@
+.home-page {
```

Comment:
add the styles to html selector
```suggestion
html {
```